packages/common/src/context.ts

```diff
@@ -13,7 +13,7 @@ export interface Context {
     ipfs?: IpfsApi; // an ipfs instance
     resolver?: Resolver; // a DID resolver instance
     provider?: DIDProvider; // a DID provider (3ID provider initially)
-    anchorService?: AnchorService;
+    anchorServices?: Record<string, Array<AnchorService>>; // mapping from chainId(s) to the AnchorService instance(s)
```

Comment:
Since we only support one anchor service per chainId currently, I think we'd be better off not trying store an array of AnchorServices here.  We're just complicating our code to handle a case that we might not implement for some time, if ever.

Reply:
Same as above.